### ISSUE TITLE
Can I have a pushpin nav bar, but on the bottom?

### ISSUE THREAD

**Issue by newpen** [Dogfalo/materialize#6076]
Is there any way to have a pushpin nav bar stayed on the bottom, and when scrolling up above a specific scroll range, the nav bar will scroll up, and the other nav bar will still on the bottom? (Just the complete opposite of the original Pushpin in the Demo)

https://materializecss.com/pushpin.html

Thanks.

**Comment by mallardduck** [comment]
Based on the docs I'd expect you should just be able to configure this yourself. I believe you'd just build the layout with the Nav at the bottom. Then setup the pushpin plugin with the top and bottom default values inverted.

https://materializecss.com/pushpin.html#options

So like:

```
top: infinity,
bottom: 0
```

Haven't tested myself, but based on the docs that's what I'd try first. Then probably just debug from there.

**Comment by newpen** [comment]
No, it doesn't make sense. The two values are "The distance in pixels from the top of the page where the element becomes fixed." and "The distance in pixels from the top of the page where the elements stops being fixed.". I don't see why exchanging the values would fix the bar on the bottom.

**Comment by mallardduck** [comment]
Wow...sorry you're right. I completely missed a word in those descriptions that changes everything.
Sorry, ignore me. 🤐
